birdhouse/default.env

```diff
@@ -30,9 +30,4 @@ export DEFAULT_CONF_DIRS='
   ./config/magpie
   ./config/twitcher
   ./config/jupyterhub
-  ./config/frontend
-  ./config/project-api
-  ./config/catalog
-  ./config/malleefowl
-  ./config/solr
```

Comment:
`ncwms2`?

Reply:
This is really odd then, because `ncwms2` was there after Autodeploy test in the previous PR !  Something else must have pulled it back in.

Reply:
Frontend was pulling `ncwms2` back in https://github.com/bird-house/birdhouse-deploy/blob/37fe32ebba87a8c6afe3aa83852bfb0dfbd1f265/birdhouse/config/frontend/default.env#L5

Reply:
Ah... ok then. Well frontend is gone in this PR too :man_shrugging: